codyze-core/src/test/kotlin/de/fraunhofer/aisec/codyze/dsl/DslTest.kt

```diff
@@ -58,13 +58,20 @@ class DslTest {
                             }
                         }
                     }
+                }
 
-                    requirements {
-                        requirement("Good Encryption") {
-                            byQuery { result -> query1(result) and query2(result) }
-                        }
+                requirements {
+                    requirement("Good Encryption") {
+                        byQuery { result -> query1(result) and query2(result) }
                     }
                 }
+
+                assumptions {
+                    accept("Assumption 1")
```

Comment:
not really a UUID 🤣

Reply:
Nop. shouldn't matter as nothing is evaluated so far but I changed it nevertheless